tools/toolchains/iar.py

```diff
@@ -89,6 +89,10 @@ def __init__(self, target, notify=None, macros=None,
         self.cppc = [main_cc]
         self.cc += self.flags["common"] + c_flags_cmd + self.flags["c"]
         self.cppc += self.flags["common"] + c_flags_cmd + cxx_flags_cmd + self.flags["cxx"]
+        if target.name in ["REALTEK_RTL8195AM"]:
+            self.cc += ["--enum_is_int"]
+            self.cppc += ["--enum_is_int"]
```

Comment:
I think we need this flag, otherwise there will be some problems caused by different enum size, for example the ticker will hang. So we are still trying to figure out a better way to do this.